app/src/org/commcare/activities/HomeNavDrawerController.java

```diff
@@ -221,6 +221,8 @@ private static int getItemIcon(String id) {
                 return R.drawable.ic_logout_nav_drawer;
             case TRAINING_DRAWER_ITEM_ID:
                 return R.drawable.ic_training_nav_drawer;
+            case UPDATE_CC_DRAWER_ITEM_ID:
+                return R.drawable.ic_cc_update_nav_drawer;
```

Comment:
we can probably use https://github.com/dimagi/commcare-android/blob/master/app/res/drawable/update_download_icon.xml instead ?

Reply:
It seems a valid option, let me see how is looks

Reply:
This is how it looks. I don't think it properly conveys the action, but with the text next it it should be fine. 
<img src="https://github.com/user-attachments/assets/1a01ebbc-a6de-4bd0-9dca-858c0a0abab4" height="400px"/>

Reply:
Actually can we use [this instead](https://github.com/dimagi/commcare-android/blob/master/app/original_vector_graphics/update_download.svg) to get rid of the background circle for consistency with other icons

Reply:
I might need to change the colour as well, let me check

Reply:
Much better
<img src="https://github.com/user-attachments/assets/3855894f-4109-4753-b4ea-055c155ab5e9" height="400px"/>

Reply:
yes, looks great!

Reply:
approved?